webapp/channels/src/components/channel_header_mobile/show_search_button/show_search_button.tsx

```diff
@@ -15,24 +13,24 @@ type Props = {
     actions: Actions;
 }
 
-export default class ShowSearchButton extends React.PureComponent<Props> {
-    handleClick = () => {
-        this.props.actions.openRHSSearch();
-    };
+const ShowSearchButton = ({ actions }: Props) => {
+    const handleClick = useCallback(() => {
+        actions.openRHSSearch();
+    }, [actions]);
```

Comment:
nit: It should be `actions.openRHSSearch`, because it is more explicit about the changes that is observing, in this case it is not a problem, because actions are normally not changing, and in this case there is only 1 action. My concern is about generating a bad usage example.

Reply:
Requested changes done. Please review it.